contrib/linux/install-icons.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+DEST_PATH="$HOME/.local/share"
+REMOVE=false
+USAGE=false
+
+strstr() {
+    [ "${1#*$2*}" = "$1" ] && return 1
+    return 0
+}
+
+usage() {
+    echo "Installs icon files and desktop shortcut."
+    echo "--system: Install to system wide directory (/usr/share)"
+    echo "--remove: Remove icons and desktop file"
+}
```

Comment:
minor: Would be a bit nicer to indent the options by 2 spaces at least?

Reply:
Sure, that looks better.  Changed.